### ISSUE TITLE
does not open

### ISSUE THREAD

**Issue by LeandroStanger** [flathub/com.anydesk.Anydesk#1]
/app/bin/anydesk: line 2: /app/extra/anydesk: Permission denied
/app/bin/anydesk: line 2: exec: /app/extra/anydesk: cannot execute: Permission denied

**Comment by nedrichards** [comment]
Yep, I can reproduce. Strange since it didn't behave that way locally. Will
try and fix later.
